src/views/Main/settings_mock.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
File should be name `settings_mock.dist.json`.

Reply:
And add a default version with default value to keep compatibility. Also add information about this file in readme.

Reply:
up

Reply:
cf /public/settings.dist.json